scanners/sslyze.py

```diff
@@ -14,8 +21,17 @@
 #
 # If data exists for a domain from `pshtt`, will check results
 # and only process domains with valid HTTPS, or broken chains.
+#
+# Supported options:
+#
+# --sslyze-serial - If set, will use a synchronous (single-threaded
+#   in-process) scanner. Defaults to false.
```

Comment:
The `--sslyze-serial` option isn't documented in the README, it was just used for debugging. In the future, I may remove the SSLyze serial code path altogether, as it generates a memory leak under normal use.